ffi/athcon/bindings/go/athcon.go

```diff
@@ -192,7 +222,8 @@ func (vm *VM) Execute(
 	code []byte,
 ) (res Result, err error) {
 	if len(code) == 0 {
-		return res, fmt.Errorf("code is empty")
+		fmt.Fprintf(os.Stderr, "athcon: no input code")
+		return res, Failure
```

Comment:
I'd prefer it not to print anything and leave it to the lib user. How about adding the context in the returned error? Like: 
```suggestion
		return res, fmt.Errorf("athcon: %w: no input code", Failure)
```